orc8r/cloud/helm/orc8r/templates/controller.deployment.yaml

```diff
@@ -92,9 +92,11 @@ spec:
               valueFrom:
                 secretKeyRef:
                   name: {{ $serviceName }}
-                  key: postgres.connstr
+                  key: {{ .Values.controller.spec.database.driver }}.connstr
             - name: SQL_DRIVER
-              value: postgres
+              value: {{ .Values.controller.spec.database.driver }}
+            - name: SQL_DIALECT
```

Comment:
What would be the purpose of this env var? Curious since we dont read it anywhere.

Reply:
https://github.com/facebookincubator/magma/blob/master/orc8r/cloud/docker/docker-compose.yml#L48